setup/upgrade_minor.rst

```diff
@@ -22,37 +22,43 @@ There are two steps to upgrading a minor version:
 ------------------------------------------
 
 The ``composer.json`` file is configured to allow Symfony packages to be
-upgraded to patch versions. But, if you would like the packages to be upgraded
-to minor versions, check that the version constrains of the Symfony dependencies
-are like this:
+upgraded to patch versions. But to upgrade to a new minor version, you will
+probably need to update the version constraint next to each library starting
+``symfony/``. Suppose you are upgrading from Symfony 4.3 to 4.4:
 
-.. code-block:: json
+.. code-block:: diff
 
     {
         "...": "...",
 
         "require": {
-            "symfony/cache": "^4.0",
-            "symfony/config": "^4.0",
-            "symfony/console": "^4.0",
-            "symfony/debug": "^4.0",
-            "symfony/dependency-injection": "^4.0",
-            "symfony/dotenv": "^4.0",
-            "...": "..."
+    -         "symfony/cache": "4.3.*",
+    +         "symfony/cache": "4.4.*",
+    -         "symfony/config": "4.3.*",
+    +         "symfony/config": "4.4.*",
+    -         "symfony/console": "4.3.*",
+    +         "symfony/console": "4.4.*",
```

Comment:
Are we sure this is the case? (I'm on Windows atm, so I cannot test it)

If I'm reading https://github.com/symfony/skeleton/blob/master/composer.json correctly, the dependencies are actually marked as `*` and the `extra.symfony.require` is used to limit the scope to a specific minor version. That would mean you only need to update that value?

Reply:
In practice, when you `compose require symfony/something`, it will put the `X.X.*` version constraint in your composer.json file. So... I'm trying to show an example that will look "like" the user's composer.json file. But technically speaking, you're totally right: if they had `^4.0` everywhere, they could upgrade/control the Symfony versions *simply* by changing `extra.symfony.require`. But that's not how things currently will look in real apps.